### PR TITLE
feat: init nuxt sdk

### PR DESCRIPTION
**Pull request by gao-sun**
## Summary

> [!Note]
> You may find the project structure, toolkit, and configuration are not similar to other packages, the reason is nuxt has many conventions (more specifically, convention over configuration), and we have to follow them. omg.

- implement workable nuxt sdk
- tested with playground

## Testing

locally tested sign-in/out (the playground project prints correct info)

## Checklist

changeset, tests, comments, etc. are coming in the following pulls.

- [ ] `.changeset`
- [ ] unit tests
- [ ] integration tests
- [ ] necessary TSDoc comments

### CODE REVIEW

**Review by wangsijie — APPROVED** [review]

File: packages/nuxt/playground/app.vue [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+<template>
+  <div>
+    <p>Logto Nuxt 3 sample</p>
+    <p v-if="Boolean(user)">Authenticated</p>
+    <ul v-if="Boolean(user)">
+      <li v-for="(value, key) in user">
+        <b>{{ key }}:</b> {{ value }}
+      </li>
+    </ul>
+    <a :href="`/sign-${ user ? 'out' : 'in' }`">
```
Comment (darcyYe) [inline packages/nuxt/playground/app.vue:15]: unnecessary colon?

File: packages/nuxt/src/module.ts [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    configKey: 'logto',
+  },
+  defaults,
+  setup(options, nuxt) {
+    // Merge runtimeConfig with module options (options already merged with defaults)
+    const runtimeConfig = defu<LogtoRuntimeConfig, LogtoRuntimeConfigInput[]>(
+      nuxt.options.runtimeConfig.logto,
+      options,
+      {
+        fetchUserInfo: false,
```
Comment (darcyYe) [inline packages/nuxt/src/module.ts:24]: what does falsy `fetchUserInfo` mean? Can authenticated users request user info if this is false?

**Review by darcyYe — APPROVED** [review]
Overall LGTM

File: packages/nuxt/playground/app.vue (hunk 1 above)
Comment (gao-sun) [reply packages/nuxt/playground/app.vue:15]: it's vue's convention. required.

File: packages/nuxt/src/module.ts (hunk 2 above)
Comment (gao-sun) [reply packages/nuxt/src/module.ts:24]: see [this comment](https://github.com/logto-io/js/pull/639/files#diff-ec13d0fe2e97302e7f9d58953bd1c95b04f3efea529b3257bc8bf17300e5bb5bR17-R23)

**Pull request merged**